h/cli/commands/search.py

```diff
@@ -67,6 +60,6 @@ def _update_settings_old(ctx):
     request = ctx.obj['bootstrap']()
 
     try:
-        config.update_index_settings(request.es)
+        config.update_index_settings(request.es6)
```

Comment:
I think this would be a good time to remove the `_update_settings_old` indirection as well.

Reply:
Just pushed this.